src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -98,6 +98,8 @@ trait Warnings {
     val Constant               = LintWarning("constant",                  "Evaluation of a constant arithmetic expression results in an error.")
     val Unused                 = LintWarning("unused",                    "Enable -Ywarn-unused:imports,privates,locals,implicits.")
     val NonlocalReturn         = LintWarning("nonlocal-return",           "A return statement used an exception for flow control.")
+    val ImplicitNotFound       = LintWarning("implicit-not-found",        "Check @implicitNotFound and @implicitAmbiguous messages.")
+    val Serial                 = LintWarning("serial",                    "@SerialVersionUID on traits and non-serializable classes.")
```

Comment:
There's something bittersweet about getting your warning put behind a lint flag. Far fewer people will see it, but it gets to be mentioned in more `.sbt` files. (Hopefully not preceded with a `-`.)

Reply:
if you can't get tpolecat mad at you, you're not doing your job.